java/arcs/core/host/AbstractArcHost.kt

```diff
@@ -60,46 +59,11 @@ typealias ParticleRegistration = Pair<ParticleIdentifier, ParticleConstructor>
  */
 @ExperimentalCoroutinesApi
 abstract class AbstractArcHost(
-    protected val coroutineContext: CoroutineContext = Dispatchers.Default,
-    updateArcHostContextCoroutineContext: CoroutineContext = Dispatchers.Default,
+    protected val coroutineContext: CoroutineContext,
+    updateArcHostContextCoroutineContext: CoroutineContext,
```

Comment:
KDoc for these as well - in fact, this could be where the primary KDoc for each lives - while in the ProdArcHost above, it could simply reference the docs of these.